extensions/http-curl/client/HTTPCallback.h

```diff
@@ -142,8 +142,8 @@ class HttpStreamingCallback final : public utils::HTTPUploadByteArrayInputCallba
       current_buffer_start_ = total_bytes_loaded_;
       current_pos_ = current_buffer_start_;
       total_bytes_loaded_ += current_vec_.size();
-      logger_->log_trace("loadNextBuffer() loaded new buffer, ptr_: %p, size: %zu, current_buffer_start_: %zu, current_pos_: %zu, total_bytes_loaded_: %zu",
-          ptr_,
+      logger_->log_trace("loadNextBuffer() loaded new buffer, ptr_: {}, size: {}, current_buffer_start_: {}, current_pos_: {}, total_bytes_loaded_: {}",
+          static_cast<void*>(ptr_),
```

Comment:
Why is the cast to `void*` needed?

Reply:
its a feature of fmt and std::format they only accept void* for printing out pointers so we dont accidentaly write
something like this: https://github.com/apache/nifi-minifi-cpp/blame/main/libminifi/src/controllers/SSLContextService.cpp#L225